src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -18,7 +18,7 @@ limitations under the License.
   <domain name="CHIP"/>
 
   Data types
-  <struct name="LocationInfoStruct" apiMaturity="provisional">
+  <struct name="AreaInfoStruct" apiMaturity="provisional">
     <cluster code="0x0150"/>
     <item name="LocationInfo"  type="LocationDescriptorStruct" optional="false" isNullable="true"/>
     <item name="LandmarkTag"   type="LandmarkTag"              optional="false" isNullable="true"/>
```

Comment:
This is pre-existing, but the spec has a LandmarkInfo of type LandmarkInforStruct, not what we have here.  Is that being addressed in a separate PR?

Reply:
Yes. It will be addressed when fixing https://github.com/project-chip/connectedhomeip/issues/34669.